### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by ThatOneGuy35** [DmACKGL/GCPhone-Rewrite#65]
**Describe the bug**
When trying to call someone, the person will not get a notification at all of the incoming call and this error appears clientside. 
SCRIPT ERROR: @gcphone/client/client.lua:37: attempt to call a nil value (local 'cb')
handler client/client.lue: 433

**To Reproduce**
Attempt to call someone

**Expected behavior**
Other person's phone to ring and give a notification

**Screenshots**

**FiVEM(please complete the following information):**
 - Artifact: Latest (as of 4/6/21)
 - Voice Software  TokoVoip

**Additional context**
None

**Comment by sparcoxalpha** [comment]
Same for me also,i already reported but no response 😞

**Comment by deBBuu** [comment]
Hi! I found an solution
Simply, go to client.lua    line 427

and replace this 
<pre>
RegisterNetEvent("gcPhone:waitingCall")
AddEventHandler("gcPhone:waitingCall", function(infoCall, initiator)
  if initiator == true then
    SendNUIMessage({event = 'waitingCall', infoCall = infoCall, initiator = initiator})
    PhonePlayCall()
    if menuIsOpen == false then
      TooglePhone()
    end
  else
    hasPhone(function (hasPhone)
      if hasPhone == true then
        SendNUIMessage({event = 'waitingCall', infoCall = infoCall, initiator = initiator})
      end
    end)
  end
end)`
</pre>
with 

<pre> 
RegisterNetEvent("gcPhone:waitingCall")
AddEventHandler("gcPhone:waitingCall", function(infoCall, initiator)
  SendNUIMessage({event = 'waitingCall', infoCall = infoCall, initiator = initiator})
  if initiator == true then
    PhonePlayCall()
    if menuIsOpen == false then
      TooglePhone()
    end
  end
end)

**Comment by sparcoxalpha** [comment]
Is there any solution for not hearing sound while phone call

**Comment by sparcoxalpha** [comment]
With tokovoip

**Comment by deBBuu** [comment]
idk

**Comment by HezimX** [comment]
> With tokovoip

Yes there is you need to add some cods in client.lua ill send you the changes when i get time

**Comment by HezimX** [comment]
> **Describe the bug**
> When trying to call someone, the person will not get a notification at all of the incoming call and this error appears clientside.
> SCRIPT ERROR: @gcphone/client/client.lua:37: attempt to call a nil value (local 'cb')
> handler client/client.lue: 433
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> Attempt to call someone
> 
> **Expected behavior**
> Other person's phone to ring and give a notification
> 
> **Screenshots**
> If applicable, add screenshots to help explain your problem.
> 
> **FiVEM(please complete the following information):**
> 
> * Artifact: Latest (as of 4/6/21)
> * Voice Software  TokoVoip
> 
> **Additional context**
> None


Fix here
https://github.com/DmACKGL/GCPhone-Rewritten/issues/66

**Comment by sparcoxalpha** [comment]
Please do asap.. waiting

**Comment by HezimX** [comment]
> Please do asap.. waiting

try this one remember to add html file 

[gcphone.zip](https://github.com/DmACKGL/GCPhone-Rewritten/files/6274669/gcphone.zip)

**Comment by ThatOneGuy35** [comment]
> Hi! I found an solution
> Simply, go to client.lua line 427
> 
> and replace this
> 
> RegisterNetEvent("gcPhone:waitingCall")
> AddEventHandler("gcPhone:waitingCall", function(infoCall, initiator)
>   if initiator == true then
>     SendNUIMessage({event = 'waitingCall', infoCall = infoCall, initiator = initiator})
>     PhonePlayCall()
>     if menuIsOpen == false then
>       TooglePhone()
>     end
>   else
>     hasPhone(function (hasPhone)
>       if hasPhone == true then
>         SendNUIMessage({event = 'waitingCall', infoCall = infoCall, initiator = initiator})
>       end
>     end)
>   end
> end)`
> with
> 
>  
> RegisterNetEvent("gcPhone:waitingCall")
> AddEventHandler("gcPhone:waitingCall", function(infoCall, initiator)
>   SendNUIMessage({event = 'waitingCall', infoCall = infoCall, initiator = initiator})
>   if initiator == true then
>     PhonePlayCall()
>     if menuIsOpen == false then
>       TooglePhone()
>     end
>   end
> end)

Doesn't work for me.

**Comment by sparcoxalpha** [comment]
> > Please do asap.. waiting
> 
> try this one remember to add html file
> 
> [gcphone.zip](https://github.com/DmACKGL/GCPhone-Rewritten/files/6274669/gcphone.zip)

Ok thanks dude,i ll try

**Comment by DmACKGL** [comment]
Hey Everyone,

This week I had a lot of work, that's why I didn't update the resource, I hope I'll return to development next week if everything goes as planned. So sorry.

All forks and pull request fixing problems are COMPLETELY welcome (and pls do, that's the idea of having an open-source project, so everyone can help to improve the resource)

(Copy paste of #67)
